mautic/mautic
# Default signature option in the configuration translated

| Q  | A
| --- | ---
| Bug fix? | Y
| New feature? | N
| Related user documentation PR URL | /
| Related developer documentation PR URL | /
| Issues addressed (#s or URLs) | /
| BC breaks? | N
| Deprecations? | N

#### Description:
Just adding a translation strings that I noticed were missing.

#### Steps to test this PR:
1. Apply this PR.
2. Repeat the test.
- The strings should be translated.

#### Steps to reproduce the bug:
1. Go to the configuration, email tab and scroll down
- The Default signature option label and tooltip are not translated.
